### PR TITLE
PXB-2701 MyRocks incremental backups (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
M  source/backup_scenarios/incremental_backup.rst
M  source/intro.rst
M  source/xtrabackup_bin/incremental_backups.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/backup_scenarios/incremental_backup.rst [hunk 1]
```diff
@@ -5,7 +5,9 @@ Incremental Backup
 ==================
 
 |xtrabackup| supports incremental backups, which means that they can copy only
-the data that has changed since the last backup.
+the data that has changed since the last backup. 
+
+.. note:: You cannot take incremental backups on the MyRocks storage engine.
```
Comment (rahulmalik87) [inline storage/innobase/xtrabackup/doc/source/backup_scenarios/incremental_backup.rst]: Can we mention something like 
In each incremental backup, PXB copies the MyRocks files again irrespective if they copied in last full/incremental backup.

Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/backup_scenarios/incremental_backup.rst]: Thank you. Rewritten.

**Review by rahulmalik87 — APPROVED** [review]
LGTM

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**